antenna/app.py

```diff
@@ -137,6 +137,21 @@ def log_config(logger, component):
         logger.info(msg)
 
 
+def build_config_manager():
+    config = ConfigManager(
+        environments=[
+            # Pull configuration from environment variables
+            ConfigOSEnv()
+        ],
+        doc=(
+            'For configuration help, see '
+            'https://antenna.readthedocs.io/en/latest/configuration.html'
+        )
+    )
+
+    return config
```

Comment:
I extracted this, but forget why. I suspect there was a reason for it, but then the reason went away. Maybe it got a job and moved to a big city somewhere. Maybe it's decided to be an astronaut. Space. Spaaaaaaaace.